edb/lib/schema.edgeql

```diff
@@ -298,6 +298,7 @@ CREATE ABSTRACT TYPE schema::Pointer
     CREATE PROPERTY default -> std::str;
     CREATE PROPERTY expr -> std::str;
     CREATE PROPERTY secret -> std::bool;
+    CREATE PROPERTY lazy -> std::bool;
```

Comment:
Can this be an enum so that we can leave the door open for marking things as eager by default?

Reply:
We can make it an enum, but I don't totally understand this suggested use case? What are the values you propose